docs/src/main/asciidoc/spring-integration-pubsub.adoc

```diff
@@ -156,7 +156,36 @@ public void messageReceiver(String payload,
 }
 ----
 
-NOTE: `AcknowledgeablePubSubMessage` objects acquired by synchronous pull are aware of their own acknowledgement IDs.
+Messages can only be retrieved one at a time from a `PubSubMessageSource`. To retrieve a batch of messages, use `PubSubBatchMessageSource`.
```

Comment:
Maybe a subsection header here for `PubSubBatchMessageSource`?

Reply:
As discussed, I think spring-gcp team is in a better place to provide documentation that meets standards.